src/main/resources/META-INF/persistence.xml

```diff
@@ -6,7 +6,7 @@
         <jta-data-source>AppDataSource</jta-data-source>
         <properties>
             <property name="hibernate.show_sql" value="true"/>
-            <property name="hibernate.hbm2ddl.auto" value="update"/>
+            <property name="hibernate.hbm2ddl.auto" value="create"/>
```

Comment:
After first launch don't forget to change it back to `value="update"`

Reply:
I don't know if we event need to use `create` as `update` seems to do the same thing 🤔